addon/components/x-file-input.js

```diff
@@ -17,9 +17,25 @@ export default Ember.Component.extend({
    * @param {$.Event} e Native change event
    */
   change(e) {
-    this.sendAction("action", e.target.files);
+    this.sendAction("action", e.target.files, this.resetInput);
```

Comment:
I think this is going to lose the context of the component. Don't you need `this.resetInput.bind(this)`